sql/core/src/main/scala/org/apache/spark/sql/execution/command/ddl.scala

```diff
@@ -180,7 +181,8 @@ case class DescribeDatabaseCommand(
       sparkSession.sessionState.catalog.getDatabaseMetadata(databaseName)
     val allDbProperties = dbMetadata.properties
     val result =
-      Row("Database Name", dbMetadata.name) ::
+      Row("Catalog Name", SESSION_CATALOG_NAME) ::
```

Comment:
seems we need to fix v2 `DescribeNamespaceExec` as well.

Reply:
yes, this pr already fix the DescribeNamespaceExec